components/LoginForm.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styles from '../styles/Login.module.css'
+
+interface LoginForm {
+  onSubmit: React.FormEvent,
+  buttonText: string,
+}
+
+function LoginForm({onSubmit, buttonText}) {

```

Comment:
You have declared an interface but you didn't typecheck it using :

Is this a convention perhaps?